go/grpc/examples/wallet/account/account.pb.go

```diff
@@ -1,28 +1,38 @@
+// Copyright 2020 Google LLC
```

Comment:
Why add the Copyright for generated code? It's not there for the other pb.go files. Also year to be 2021?

Reply:
The Copyright notice is part of the `.proto` files, and the codegen simply copies it over to here.

It is present in all `*.pb.go` files. It is only absent in `*_grpc.pb.go` files. The latter contains generated code for services while the former contains generated code for messages. They are both in the same go package. Hence the comments from the corresponding proto are copied over only to one and not both of them.